tests/rt.sh

```diff
@@ -218,7 +218,7 @@ elif [[ $MACHINE_ID = orion.* ]]; then
 
   module use $PATHTR/modulefiles/${MACHINE_ID}
   module load fv3
-  module load gcc/8.3.0
+  #module load gcc/8.3.0
```

Comment:
In Dusan's email, he said to not remove the ccc/8.3.0 module from rt.sh, so is this something we want to do?

Reply:
Once the stochastic_physics is updated with the makefile changes, the line "module load gcc/8.3.0" will be used again.
I will update the stochastic_physics submodule before the merge to the s2s develop. The baselines will not change.